cf-agent/verify_processes.c

```diff
@@ -246,7 +246,7 @@ static PromiseResult VerifyProcessOp(EvalContext *ctx, Attributes a, const Promi
         else
         {
             PromiseResult status = killed ? PROMISE_RESULT_CHANGE : PROMISE_RESULT_NOOP;
-            cfPS(ctx, LOG_LEVEL_INFO, status, pp, a,
+            cfPS(ctx, LOG_LEVEL_VERBOSE, status, pp, a,
```

Comment:
Verbose is better I agree. But message should contain the `C:` string like other class settings.